motivaa-backend/motivaa-app/src/test/java/backendtesting/RequestMaker.java

```diff
@@ -0,0 +1,66 @@
+package backendtesting;
+
+import io.restassured.RestAssured;
+
+import java.util.Map;
+
+public class RequestMaker {
+
+    public static String initiateGetRequest(String host,
+                                            String endpoint,
+                                            String jSessionId,
+                                            boolean shouldRequestBeLogged) {
+        if (shouldRequestBeLogged) {
+            return RestAssured.given()
+                    .log().all()
+                    .cookie("JSESSIONID", jSessionId)
+                    .post(host+endpoint)
+                    .then()
+                    .statusCode(200)
```

Comment:
@cynran , do you think I should validate the JsonSchema too? I could generate it from a response with a tool like https://www.liquid-technologies.com/online-json-to-schema-converter and validate it, however as I don't own the schema I am not sure it would be recommended. What is your view?

Reply:
I meant this: https://github.com/rest-assured/rest-assured/wiki/Usage#json-schema-validation

What I was thinking:
1. grab a response value by sample
2. generate schema from it
3. validate if schema is the same

I could imagine that maybe this would be more a test case for the API itself and not the tested functionality (excels on modules here). On the other side, I could argue that its testing if this part works well in the context of excels on modules.

but I could also argue that until I get the expected result back maybe I shouldnt care at all. I felt that I spend time thinking about it but I am not arriving to conclusion, hence I you.